### DISCUSSION TITLE
Moving existing production repositories to TF

### DISCUSSION THREAD

**Discussion by nrowealign** [jfrog/terraform-provider-artifactory#586] [Q&A] [answered]
Hello,

We're very keen to move our very large (300TB) instance to using this project to manage configuration.

I've written some code to query the REST API and spit out TF syntax, but when I `terraform plan` the plan indicates that existing repositories will be created as new. This will prevent us from ensuring that we've correctly codified the existing setup in TF which will in turn prevent us from using this project.

Is there something I'm doing wrong?

We're using Terraform v1.2.5 with this provider v6.20.1. Thanks!

**Comment by alexhung** [comment, marked as answer]
I may already have answered this in the Issues section, in order to use TF to manage *existing* resources, you need to import those resources once you have defined the configuration.
